### ISSUE TITLE
Bug for multisite with fix

### ISSUE THREAD

**Issue by comfortpay** [blockonomics/woocommerce-plugin#5]
Hi,

In the file blockonomics-woocommerce you use a wrong code to check if woocommerce is active. Its not really your fault as you use the official code from the woocommerce docs, but their code is outdated as it doesnt work on multisite networks.

In order to check if woocommerce is active you need to replace this line:

if ( in_array( 'woocommerce/woocommerce.php', apply_filters( 'active_plugins', get_option( 'active_plugins' ) ) ) ) {

with this:

if ( class_exists( 'WooCommerce' ) ) {

as mentioned here: http://wordpress.stackexchange.com/questions/159177/how-to-check-if-woocommerce-is-activated-in-theme/159179

This way your plugin also works on multisite networks 

cheers

**Comment by blockonomics** [comment]
Thanks.. really appreciate you helping us out here. Will officially commit and test in few days

**Comment by comfortpay** [comment]
you are welcome :)

**Comment by shivaenigma** [comment]
Using is_plugin_active as class_exists( 'WooCommerce' )  is not working for me https://wp-buddy.com/blog/check-wordpress-plugin-sitewide-active-network-active/

**Comment by comfortpay** [comment]
yeah I had the same, seems the official code is outdated. Below seems to work in all cases, its not the official way but woocommerce uses this code in its own plugins as well

if ( class_exists( 'WooCommerce' ) ) {

**Comment by comfortpay** [comment]
Hi,

I had 2 bitcoin payments today. But for some reason I had to manually adjust them to “complete” in woocommerce as the order kept staying on “processing”. 
The time limit of the payment exeeded but that happens a lot in bitcoin payments.

Any idea why the orders where not fully processed after payment?

I do use a plugin called “autocomplete” orders. I can try to deactivate that to see if that makes a difference.

Attached the screenshots of the 2 payments

ps. de second screenshot you see its fully processed, but that is because I manually changed it to complete status

cheers

Robin
